spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -869,6 +869,20 @@ a non-case-sensitive matching. The `IgnoreCase` phrase may only be appended to f
 to columns of type STRING or BYTES.
 The Spring Data "AllIgnoreCase" phrase appended at the end of the method name is not supported.
 
+The `Like` or `NotLike` naming conventions:
```

Comment:
Can we document all of the keywords that we support like the table here? https://docs.spring.io/spring-data/data-jpa/docs/1.3.4.RELEASE/reference/html/repository-query-keywords.html

Reply:
Yes please make an issue of it